bugbug/repository.py

```diff
@@ -241,24 +241,47 @@ def _transform(commit):
             type_ = "Rust"
         else:
             type_ = ext
-        commit.types.add(type_)
 
+        size = None
         if not stats["deleted"]:
             try:
                 after = HG.cat([path.encode("utf-8")], rev=commit.node.encode("ascii"))
-                sizes.append(after.count(b"\n"))
+                size = after.count(b"\n")
             except hglib.error.CommandError as e:
                 if b"no such file in rev" not in e.err:
                     raise
 
+        if is_test(path):
+            commit.test_files_modified_num += 1
+
+            commit.test_added += stats["added_lines"]
+            commit.test_deleted += stats["deleted_lines"]
+
+            if size is not None:
+                test_sizes.append(size)
+        else:
+            commit.files_modified_num += 1
+
+            commit.added += stats["added_lines"]
+            commit.deleted += stats["deleted_lines"]
+
+            if size is not None:
+                sizes.append(size)
+            commit.types.add(type_)
```

Comment:
Is it on purpose to not add type for a test ?

Reply:
Yeah, I initially added it, but then I removed it as almost all tests are JS, so I think there isn't much info in this.

Reply:
ok, just add a comment to not forget why we don't add them

Reply:
Done.